_posts/2020-07-14-newpipe-0.19.6-release.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "NewPipe 0.19.6 released: Audio/video desync bug and search improvements"
+short: "NewPipe 0.19.6 released"
+date: 2020-07-14 00:00:00 +0100
+categories: [release, pinned]
+author: gkeegan
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+No more unsynced videos! This release will let your videos make sense and improve searching in NewPipe to make it more reliable and widespread.
+
+<!-- more -->
+
+## Additions
+
+You can now add a playlist directly to your main page as a tab, a great change for those who frequently use NewPipe for music. <sup>[@Royosef in #3506](https://github.com/TeamNewPipe/NewPipe/pull/3506)</sup>
+
+There is an option to prevent subscriptions from being visible in the subscription picker for feeds which will be helpful to sort your subscriptions into different categories. <sup>[@mauriciocolli in #3404](https://github.com/TeamNewPipe/NewPipe/pull/3404)</sup>
+
+Speaking (writing) of the subscription picker, it now has a search bar so you can easily find what you are looking for. <sup>[@mauriciocolli in #3373](https://github.com/TeamNewPipe/NewPipe/pull/3373)</sup>
+
+More search changes include "did you mean" & "showing result for" suggestions from YouTube so that you don't have to restart a search if you make a tpyo. <sup>[@Royosef in #3471](https://github.com/TeamNewPipe/NewPipe/pull/3471)</sup>
+
+When using the background player, you can fast forward and rewind through the playback from within the menu for it (not through the notification). <sup>[@TheLastGimbus in #3473](https://github.com/TeamNewPipe/NewPipe/pull/3437)</sup>
+
+PeerTube videos are now able to be opened in NewPipe without explicitly sharing to it. The [privacytools.io](https://www.privacytools.io/) PeerTube instance was added to NewPipe. PeerTube embed links are also now able to be opened in NewPipe now. <sup>[@B0pol in #3406](https://github.com/TeamNewPipe/NewPipe/pull/3406), in [#3845](https://github.com/TeamNewPipe/NewPipe/pull/3845), and in [Extractor #344](https://github.com/TeamNewPipe/NewPipeExtractor/pull/344)</sup>
+
+More Invidious instances are added to the app manifest, so NewPipe will be suggested as an application if you open these links. <sup>[@B0pol in #3841](https://github.com/TeamNewPipe/NewPipe/pull/3841)</sup>
```

Comment:
An average user doesn't know what the manifest is

Reply:
How about this?

```suggestion
More Invidious instances are supported for link opening so that NewPipe will be suggested as an application if you click those links. <sup>[@B0pol in #3841](https://github.com/TeamNewPipe/NewPipe/pull/3841)</sup>
```

Reply:
Hmm… "for link opening" and "if you click those links" is kinda double

Reply:
Agreed. This seems better.
```suggestion
More Invidious instances are supported so that NewPipe will be suggested as an application if you open these links. <sup>[@B0pol in #3841](https://github.com/TeamNewPipe/NewPipe/pull/3841)</sup>
```